src/marqo/core/document/document.py

```diff
@@ -57,7 +57,7 @@ def partial_update_documents_by_index_name(self, index_name,
         marqo_index = self.index_management.get_index(index_name)
 
         res: MarqoUpdateDocumentsResponse = self.partial_update_documents(partial_documents, marqo_index)
-        return res.dict(exclude_none=True)
+        return res.dict(exclude_none=True, by_alias=True)
```

Comment:
Wouldn't it be nicer to return the model rather than a dict?

Reply:
Done. We now return the model directly and format the response in the API function.